### PR TITLE
dev_987 Adjusting Mortgage

### PR DESCRIPTION
**Pull request by vegot95**
Not bad for the first try

### CODE REVIEW

**Comment by EldarKafarov** [comment]
Okay, I will review this

**Comment by EldarKafarov** [comment]
Are you ready for my review

**Comment by rkocis08** [comment]
give me a few days

File: notes [hunk 1]
```diff
@@ -13,3 +13,4 @@ Agenda:
 			- git pull origin master
 			- switch back to your own branch git checkout {your branch}
             - git rebase origin/master
+Gulalek was here
```
Comment (rkocis08) [inline notes:16]: who's not cooking

**Review by EldarKafarov — COMMENTED** [review]
Good to have you at mindtek

**Comment by iu-fedotov** [comment]
+iurii is here =)

File: notes [hunk 2]
```diff
@@ -13,3 +13,4 @@ Agenda:
 			- git pull origin master
 			- switch back to your own branch git checkout {your branch}
```
Comment (EldarKafarov) [inline notes:14]: Line 14 needs attention

File: notes (hunk 1 above)
Comment (iu-fedotov) [inline notes:16]: I'm not =(

File: notes (hunk 2 above)
Comment (EldarKafarov) [reply notes:14]: Done, it was fixed

**Comment by EldarKafarov** [comment]
LGTM

**Pull request merged**